seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/ExcelReadStrategy.java

```diff
@@ -215,7 +215,7 @@ private Object convert(Object field, SeaTunnelDataType<?> fieldType) {
             case ARRAY:
                 return objectMapper.readValue((String) field, fieldType.getTypeClass());
             case STRING:
-                return field;
+                return String.valueOf(field);
```

Comment:
Thanks @CosmosNi , please add test case in https://github.com/apache/seatunnel/tree/c685c5c71dd29618859e2f52fe552333b00729e8/seatunnel-connectors-v2/connector-file/connector-file-base/src/test/java/org/apache/seatunnel/connectors/seatunnel/file/writer